src/Service/Math.php

```diff
@@ -28,10 +28,10 @@ public static function isZero($number, $precission = 2)
      */
     public static function netToGross($netPrice, $taxPercentage, $round = false)
     {
-        $withTax = $netPrice + $netPrice * $taxPercentage / 100;
+        $withTax = $netPrice * (1 + $taxPercentage / 100);
 
         if (!$round) {
-            return $withTax;
+            return floatval(number_format($withTax, 5));
         }
```

Comment:
I have tested this with your changes and it provides incorrect values now. 
Method: netToGross($netPrice, $taxPercentage) 
Test-1:
    netToGross(100, 19);
     This returns 119 which is correct.

Test-2     
    netToGross(10000, 19);
     This returns 11 which is not correct, it should return 11900
     
    Please test it again with higher values. Also update the MathTest.php test and add data set with high amount to avoid miss calculations.

Reply:
i can reproduce the issue, which @hamail-tahir got. 

but i would replace this with the following to prevent using local server settings. 
Floats always have the `.` as decimal separator.

```floatval(number_format($withTax, 5, '.', ''));```